api/Node.json

```diff
@@ -0,0 +1,1107 @@
+{
+  "api": {
+    "Node": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node",
+        "support": {
+          "webview_android": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
Chrome Android should match Webview Android.

Reply:
👍